playbooks/hc-ansible-deployment/tasks/replace_node.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: hc_nodes
+  remote_user: root
+  gather_facts: yes
+  tasks:
+    - block:
+      - name: Gluster host preparation
+        include_role:
+         name: "{{ item }}"
```

Comment:
When I executed the playbook, I got this warning - "The loop variable 'item' is already in use." for each task execution. I have replaced it with separate loop_variable and that solved the problem. Please incorporate the same